release_notes/41-release-notes.adoc

```diff
@@ -393,6 +393,7 @@ When Central was reinstalled, the password of the new secret didn't match the on
 [id="known-issues_{context}"]
 === Known issues
 
+* For secured clusters on {ibm-z} and {ibm-linuxone} that are running {ocp} 4.10, Collector fails to start using the default configuration. As a workaround, enable the xref:../release_notes/41-release-notes.adoc#new-collection-method-based-on-bpf-co-re-technology-preview_release-notes-41[CORE_BPF collection method] (Technology Preview). (ROX-18058)
```

Comment:
To me this could read two ways:

* it fails to start up when you're using the default configuration
* it fails to begin using this configuration

If it's the former, maybe some tweak to clarify, like "Collector fails to start when the default configuration is used.". If it's the latter, then some other tweak that I can't give a suggestion for :smile:

Reply:
Thanks - it's #1, so I will adjust accordingly!